src/global-messages/GlobalMessage.tsx

```diff
@@ -60,39 +60,37 @@ const GlobalMessage = ({
         .filter((gm) => isWithinTimeRange(gm, now))
         .map((globalMessage: GlobalMessageType) => {
           const message = getTextForLanguage(globalMessage.body, language);
+          const onPressAction = globalMessage.link
+            ? {text: globalMessage.link, url: globalMessage.link}
+            : undefined;
```

Comment:
I think we should allow to set both link url and text for it through Firestore. The link might be long and non-presentable.